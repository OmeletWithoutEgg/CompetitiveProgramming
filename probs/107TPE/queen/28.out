-478178.06 18250005.07
