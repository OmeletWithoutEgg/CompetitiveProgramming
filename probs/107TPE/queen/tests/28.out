-836789.82 -36700524.99
